src/app/ui/components/flag/flag.tsx

```diff
@@ -33,13 +34,23 @@ interface FlagProps extends FlexProps {
 export function Flag({
   spacing = 'space.03',
   align = 'middle',
+  reverse = false,
   img,
   children,
   ...props
 }: FlagProps) {
   return (
-    <Flex flexDirection="row" width="100%" data-align={align} className={flagStyles} {...props}>
-      <Box mr={spacing}>{img}</Box>
+    <Flex
+      flexDirection={reverse ? 'row-reverse' : 'row'}
+      data-reversed={reverse}
```

Comment:
Why the data attribute? It seems unused?

Reply:
Removed, was going to build it using a data-* first